### PR TITLE
bug 1688487 moving monitoring

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1688487

@liangxia, will you PTAL?

### CODE REVIEW

**Comment by liangxia** [comment]
@juzhao will review the changes.

File: modules/infrastructure-moving-monitoring.adoc [hunk 1]
```diff
@@ -5,10 +5,65 @@
...
-The monitoring solution uses Cluster Version Operator (CVO) to create the
-ConfigMap that the monitoring Operator uses to determine how to deploy its
-resources. Because it uses the CVO and users cannot modify the CVO, you cannot
-change where the Operator deploys resources.
-====
\ No newline at end of file
+
+By default, a Prometheus, Grafana, and AlertManager stack is deployed to
+provide cluster monitoring. It is managed by the Cluster Monitoring Operator,
+and to move its components to different machines, you apply a custom ConfigMap.
```
Comment (juzhao) [inline modules/infrastructure-moving-monitoring.adoc]: I think it should be
change "stack is" to "stacks are"
"It is managed" to "they are managed"
"you apply" to "you should apply"

File: modules/infrastructure-moving-monitoring.adoc [hunk 2]
```diff
@@ -5,10 +5,65 @@
...
+
+.Procedure
+
+. Optionally, review the default ConfigMap for the monitoring stack:
++
+----
+$ oc get configmap cluster-monitoring-config -n openshift-monitoring
```
Comment (juzhao) [inline modules/infrastructure-moving-monitoring.adoc]: no need for this step, cluster-monitoring-config configmap is not exist when the cluster is UP, we could create the configmap by the following yaml file or edit the cluster-monitoring-config configmap if it is created

File: modules/infrastructure-moving-monitoring.adoc [hunk 3]
```diff
@@ -5,10 +5,65 @@
...
+        node-role.kubernetes.io/infra: ""
+----
++
+Running this ConfigMap forces the components of the monitoring stack to redeploy
+to infrastructure nodes.
+
+. Apply the new ConfigMap:
++
+----
+$ oc create cluster-monitoring-configmap.yaml
```
Comment (juzhao) [inline modules/infrastructure-moving-monitoring.adoc]: should be "oc create -f cluster-monitoring-configmap.yaml"

Note: we assume the cluster-monitoring-config configmap is not exist

File: modules/infrastructure-moving-monitoring.adoc [hunk 4]
```diff
@@ -5,10 +5,65 @@
...
+----
+
+. Watch the monitoring Pods move to the new machines:
++
+----
+$ watch 'oc get pod -n openshift-monitoring'
```
Comment (juzhao) [inline modules/infrastructure-moving-monitoring.adoc]: 'oc get pod -n openshift-monitoring -o wide'
would be better

File: modules/infrastructure-moving-monitoring.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/infrastructure-moving-monitoring.adoc]: It's one stack that contains multiple components, so it's right to use singular here. I'll make that clearer.

**Comment by kalexand-rh** [comment]
This bug was approved on the PR. Merging.

**Pull request merged**